spec/features/deleting_car_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+feature 'Deleting a car' do
+  scenario 'delete a car' do
+    FactoryGirl.create(:car)
+
+    visit root_path
+
+
+    click_link 'destroy_car_1'
+    # save_and_open_page
+
```

Comment:
How about an expectation here for your text, to not except to find some info about the car on the list of cars now?